### PR TITLE
Fixed broken titles and links

### PR DESCRIPTION
**Pull request by Dassine**
The google and apple links were broken as well as the indentation style of some titles. Thanks for reviewing it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,18 +22,18 @@ Methods should look like this:
...
 }
 ```
 
 If you have more than one parameter, giving each its own line is preferred. If multiple lines are used, align each using the colon before the parameter.
 
 ```objective-c
 - (void)doSomethingWith:(GTMFoo *)theFoo
-                   rect:(NSRect)theRect
-               interval:(float)theInterval
+rect:(NSRect)theRect
```
Comment (shiki) [inline README.md:33]: It looks like this and a few other lines' indentations were accidentally changed. Were these intentional?

**Review by shiki — CHANGES_REQUESTED** [review]
Thank you for contributing! If you don't mind fixing the indentations, we can take a look again 🙂

**Comment by Dassine** [comment]
@shiki unfortunately I don't have access to my branch :)

**Comment by shiki** [comment]
@Dassine No worries. I'm closing this and applied your commits on a separate PR: #3.

**Pull request closed without merging**